middleware/validation.js

```diff
@@ -240,3 +240,9 @@ export const validateArticleId = [
     .isUUID()
     .withMessage('Please provide a valid id for the article')
 ];
+
+export const validateImages = [
```

Comment:
Are you checking for maximum image size and that the image is actually an image?

Reply:
Yes, I took those factors into consideration and handled it in a separate middleware.

Reply:
Great!